packages/inventory/src/migrations/run-migration.ts

```diff
@@ -0,0 +1,63 @@
+import { InternalModuleDeclaration, LoaderOptions } from "@medusajs/modules-sdk"
+import { DataSource, DataSourceOptions } from "typeorm"
+import { InventoryServiceInitializeOptions } from "../types"
+
+import migrations from "./index"
+
+function getDataSource(
+  dbData: InventoryServiceInitializeOptions["database"]
+): DataSource {
+  return new DataSource({
```

Comment:
suggestion: Like in the core databaseLoader, export from the file a `const datasource` as it must be globally available if you want to add repositories in the future. Wdyt? That way if someone else try to add a repo, he will not have to think about it :)

Reply:
I think didn't get your point. This is only connecting to run the migration.

Reply:
If a module needs to create new custom repository, which is something we should do to avoid having the query builders in utilities or services, it needs to access the dataSource. Like in the core, it has to be exposed globally. You can look what I ve done in the core database loader

Reply:
let's keep it that way for now and see later for my comment